doc/undefined-behavior.md

```diff
@@ -0,0 +1,67 @@
+This includes at least behaviors 12, 13, 18-21, 36, 46, 61, 62.
+
+Further behaviors refer to types that are not available or semantically meaningful at the binary level, such as `void`. This includes at least behavior 22.
+
+The following table lists the remaining behaviors. It is far from complete, and we hope to expand it over time.
+
+<!-- TODO(#85): The remaining rows -->
+<!-- TODO(#85): Investigate "?"s -->
+
+| Number  | Description | Checked by GREASE | Notes      |
+| ------- | ----------- | ----------------- | ---------- |
+| 5 | The execution of a program contains a data race (5.1.2.5). | No | See [limitations](limitations.md) |
+| 9 | An object is referred to outside of its lifetime (6.2.4). | Yes | See [Memory model](memory-model.md) |
+| 10 | The value of a pointer to an object whose lifetime has ended is used (6.2.4). | Yes | See [Memory model](memory-model.md) |
+| 11 | The value of an object with automatic storage duration is used while the object has an indeterminate representation (6.2.4, 6.7.11, 6.8). | Yes | See [Memory model](memory-model.md) |
+| 16 | Conversion to or from an integer type produces a value outside the range that can be represented (6.3.1.4). | No | Types are not available at the binary level and all conversions are well-defined |
+| 17 | Demotion of one real floating type to another produces a value outside the range that can be represented (6.3.1.5). | ? | Needs test case |
+| 23 | Conversion of a pointer to an integer type produces a value outside the range that can be represented (6.3.2.3). | Somewhat | Conversions themselves are not checked, as types are not available. Some forms of pointer arithmetic raise errors. |
+| 24 | Conversion between two pointer types produces a result that is incorrectly aligned (6.3.2.3). | ? | Needs test: [#88](https://github.com/GaloisInc/grease/issues/88) |
+| 25 | A pointer is used to call a function whose type is not compatible with the referenced type (6.3.2.3). | No | Types are not available at the binary level. |
+| 32 | The program attempts to modify a string literal (6.4.5). | Yes | Such literals end up in read-only memory in the binary. |
+| 34 | A side effect on a scalar object is unsequenced relative to either a different side effect on the same scalar object or a value computation using the value of the same scalar object (6.5.1). | ? | |
+| 35 | An exceptional condition occurs during the evaluation of an expression (6.5.1). | ? | |
+| 37 | A function is defined with a type that is not compatible with the type (of the expression) pointed to by the expression that denotes the called function (6.5.3.3). | No | Types are not available at the binary level. |
+| 38 | A member of an atomic structure or union is accessed (6.5.3.4).	 | ? | |
+
+## Exceptions
+
+Certain operations generate traps or exceptions at the processor level. GREASE attempts to treat these as errors. These include:
+
+- Division by zero
```

Comment:
@RyanGlScott Is there anything else we should list here? Floating point exceptions, perhaps?

Reply:
Macaw is not very well equipped to reason about properties of floating-point numbers, so I'm doubtful that `grease` would be able to find much (if anything) there.